src/less/section-title/base/section-title.less

```diff
@@ -0,0 +1,223 @@
+/* stylelint-disable no-descending-specificity */
```

Comment:
Let's remove this if we can. As this is a new module, without any past baggage, we should aim for 0 violations of this rule.

Reply:
@ianmcburnie I ran into below lint error, and I didn't know how to resolve without duplicating some styling. I tried re-ordering, but it raises a similar linting error.

Error:
```
src/less/section-title/base/section-title.less
 42:1  ✖  Expected selector ".section-title__cta > a" to come before selector                                         no-descending-specificity
          ".section-title__cta.section-title__cta--no-text > a"                                                                                
 43:1  ✖  Expected selector ".section-title__title > h2 > a" to come before selector                                  no-descending-specificity
          ".section-title__cta.section-title__cta--no-text > a"                                                                                
 52:5  ✖  Expected selector ".section-title__cta > a:visited" to come before selector ".section-title__title > h2 >   no-descending-specificity
          a:hover"
```

CSS it's complaining about:
```
.section-title__cta > a,
.section-title__title > h2 > a {
    color: @color-black;
    text-decoration: none;

    &:hover {
        color: @color-black;
        text-decoration: underline;
    }

    &:visited {
        color: @color-black;
    }
}
```

Reply:
I tried this now and ran into similar issue (see below error). I think issue is with ` > h2` part. I'll separate out the stylings and remove `/* stylelint-disable no-descending-specificity */`. Duplicate styling is minimal.

```
src/less/section-title/base/section-title.less
 54:5  ✖  Expected selector ".section-title__cta > a:visited" to come before selector ".section-title__title > h2 >   no-descending-specificity
          a:hover
```

Reply:
I ended up adding a new class `section-title__title-container` and moved `section-title__title` on the `h2`. Now I no longer ignore the lint rule and selectors are simpler.